DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java

```diff
@@ -25,7 +33,9 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 
 public class DhsTranslatorTest {
+    private static final Integer numberOfThreads = 100;
     private final static String BASE_URI = "http://localhost:9090/axis2/services/dhs/images";
+    private static boolean isNotOk = false;
```

Comment:
I'm not sure this is guaranteed to be correct in a multi-threaded environment but regardless this flag shouldn't be needed.  You can just let the `Callable` fail with an exception and let that be your guide that something went wrong when you get the result from the `Future`.